### PR TITLE
Flag for manual review messaging

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

These changes address https://github.com/department-of-veterans-affairs/va.gov-team/issues/6859

The messages don't link to any external documentation since the details are already provided as part of the message body.  The content has been modeled after the [VA error message guidelines](https://design.va.gov/patterns/messaging-error-messages#additional-guidance)

## Screenshots

An example:

image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -91,7 +91,17 @@ jobs:
       - image: circleci/node:10.18
     environment:
       CODE_PATTERN: (/* eslint-disable)|(// eslint-disable)
-      OVERALL_REVIEW_COMMENT: ESLint is disabled - _please_ wait for a VSP review.
+      OVERALL_REVIEW_COMMENT: >
+        # ESLint is disabled
+
+        `vets-website` uses ESLint to help enforce code quality. In most
+        situations we would like for to remain enabled.
```
Comment (kristinagore) [inline .circleci/config.yml:98]: Change second sentence to "In most situations we would like ESLint to remain enabled."

**Review by kristinagore — APPROVED** [review]
I suggested one change to a sentence, but otherwise looks good!

File: .circleci/config.yml (hunk 1 above)
Comment (bkjohnson) [reply .circleci/config.yml:98]: Oops, thank you for the reminder!  I made a typo when I meant to remove the `for` instead of the `it`.

**Review by U-DON — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -123,7 +144,18 @@ jobs:
       - image: circleci/node:10.18
     environment:
       CODE_PATTERN: (<i )|(<i$)
-      OVERALL_REVIEW_COMMENT: Found `<i>` tag in changes. _Please_ wait for a VSP review.
+      OVERALL_REVIEW_COMMENT: >
+        # Icon found
+
+        Icons can be decorative, but sometimes they are used to convey meaning.
```
Comment (rianfowler) [inline .circleci/config.yml:150]: @1Copenut is there something we can link to that elaborates on this? maybe in the design system?

Comment (1Copenut) [reply .circleci/config.yml:150]: @rianfowler I looked through design.va.gov and there's not anything specific to this point (though there should be). What usually happens with the `<i>` icons being flagged, they exist without an `aria-hidden="true"` attribute. JAWS treats them like a tab stop, but doesn't announce them, which is confusing. As long as they have descriptive text nearby, they are decorative and should be hidden.

**Comment by bkjohnson** [comment]
I'm going to merge this for some immediate relief to the ongoing frustrations - but I'd like to keep gathering feedback so we can improve on this.

**Pull request merged**